openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -1372,18 +1414,36 @@ def check_handler_compatibility(self,
         #self._long_range_dispersion:'long_range_dispersion'
         #}
 
-    # TODO: nonbondedMethod and nonbondedCutoff should now be specified by StericsForce attributes
     def create_force(self, system, topology, **kwargs):
 
+        self._validate_parameters()
+
+        # If there's no cutoff, make sure the system is nonperiodic/vacuum
+        if (self._long_range_dispersion == "None") and (not topology.is_vacuum):
+            raise SMIRNOFFSpecError("If vdW long_range_dispersion is None, a vacuum/nonperiodic Topology "
+                                    "must be provided")
+        # If there is a cutoff, make sure the system is periodic/condensed
+        if (self._long_range_dispersion != "None") and (not topology.is_condensed):
+            raise SMIRNOFFSpecError("If vdW long_range_dispersion is isotropic or LJPME, a condensed/periodic Topology "
+                                    "must be provided")
+
+
         force = openmm.NonbondedForce()
-        nonbonded_method = self._NONBOND_METHOD_MAP[self._nonbonded_method]
-        force.setNonbondedMethod(nonbonded_method)
-        force.setCutoffDistance(self._cutoff.in_units_of(unit.nanometer))
-        if 'ewaldErrorTolerance' in kwargs:
-            force.setEwaldErrorTolerance(kwargs['ewaldErrorTolerance'])
-        if 'useDispersionCorrection' in kwargs:
-            force.setUseDispersionCorrection(
-                bool(kwargs['useDispersionCorrection']))
+        # If we're using PME, then the only possible openMM Nonbonded type is LJPME
+        if self._long_range_dispersion == 'LJPME':
+            force.setNonbondedMethod(openmm.NonbondedForce.LJPME)
+            force.setCutoffDistance(self._cutoff.in_units_of(unit.nanometer))
+        elif self._long_range_dispersion == 'isotropic':
+            force.setNonbondedMethod(openmm.NonbondedForce.PME)
```

Comment:
Are we sure that the electrostatics treatment here has also selected `PME`?

Reply:
So, this is resolved using a lazy hack for now. 

The Electrostatics handler has the vdW handler as a dependency, so we know that the vdW handler (this code) will run first. The Electrostatics handler them checks the openMM system for whether LJPME is set. If it is, the electrostatics handler [ensures that its `method` is set to PME](https://github.com/openforcefield/openforcefield/blob/54682abe6af67636e6b1286e08feadc30e77725e/openforcefield/typing/engines/smirnoff/parameters.py#L1584-L1592). The ugly part is that the communication hard-coded in a way it shouldn't be. We'll be able to remove that when we separate electrostatics and vdW into separate openMM forces.